Cpp_hands_on_programming/lib/other/Enum.h

```diff
@@ -0,0 +1,20 @@
+#ifndef ENUM
+#define ENUM
+
+typedef enum {
+	PIPES,
+	QUEUE,
+	SHM,
+	NONE
+} methods;
+
+typedef enum {
+    RECEIVE,
+    SEND
+} send_or_receive;
+
+typedef enum{
```

Comment:
why  are you using enum to store data size?

Reply:
I use it as constant value for data size, should I use const int instead?